src/components/label/label.wp.scss

```diff
@@ -56,6 +61,11 @@ $label-wp-text-color-focused:         color($colors-wp, primary) !default;
   transform: translate3d(0, 0, 0) scale(.8);
 }
 
+html[dir="rtl"] .input-has-focus .label-wp[floating],
```

Comment:
actually it's needed as the [dir="rtl"] .label-wp[floating] transform: translate3d will replace the .input-has-focus .label-wp[floating] on ltr